xgboost/README.md

```diff
@@ -0,0 +1,107 @@
+# Ames housing value prediction using XGBoost on Kubeflow
+
+In this example we will demonstrate how to use Kubeflow with XGBoost using the [Kaggle Ames Housing Prices prediction](https://www.kaggle.com/c/house-prices-advanced-regression-techniques/). We will do a detailed
+walk-through of how to implement, train and serve the model. You will be able to run the exact same workload on-prem and/or on any cloud provider. We will be using [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine/) to show how the end-to-end workflow runs on [Google Cloud Platform](https://cloud.google.com/). 
+
+# Pre-requisites
+
+As a part of running this setup on Google Cloud Platform, make sure you have enabled the [Google
+Kubernetes Engine](https://cloud.google.com/kubernetes-engine/). In addition to that you will need to install
+[Docker](https://docs.docker.com/install/) and [gcloud](https://cloud.google.com/sdk/downloads). Note that this setup can run on-prem and on any cloud provider, but here we will demonstrate GCP cloud option. 
+
+# Steps
+ * [Kubeflow Setup](#kubeflow-setup)
+ * [Data Preparation](#data-preparation)
+ * [Dockerfile](#dockerfile)
+ * [Model Training](#model-training)
+ * [Model Export](#model-export)
+ * [Model Serving Locally](#model-serving)
+ * [Deploying Model to Kubernetes Cluster](#deploying-the-model-to-kubernetes-cluster)
+
+## Kubeflow Setup
+In this part you will setup Kubeflow on an existing Kubernetes cluster. Checkout the Kubeflow [setup guide](https://github.com/kubeflow/kubeflow#setup).  
+
+### Requirements
+
+ * ksonnet
+ * Kubernetes
+ * minikube
```

Comment:
I don't see any references to minikube or hyperkit here, but if it's not covered in the Kubeflow setup guide, either add it there or expand a bit on it here.

Reply:
Done

Reply:
GKE training done and now working on serving K8s.